opencti-platform/opencti-graphql/src/modules/case/case-incident/case-incident-types.ts

```diff
@@ -0,0 +1,35 @@
+import type { BasicStoreEntity, StoreEntity } from '../../../types/store';
+import type { StixDomainObject, StixOpenctiExtensionSDO } from '../../../types/stix-common';
+import { STIX_EXT_OCTI } from '../../../types/stix-extensions';
+
+export const ENTITY_TYPE_CONTAINER_CASE_INCIDENT = 'Case-Incident';
+
+export interface BasicStoreEntityCaseIncident extends BasicStoreEntity {
```

Comment:
Miss severity, priority and incidentResponseType properties

Reply:
✅